src/NzbDrone.Core/Notifications/MediaBrowser/MediaBrowser.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using FluentValidation.Results;
+using NzbDrone.Common;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Core.Tv;
+
+namespace NzbDrone.Core.Notifications.MediaBrowser
+{
+    public class MediaBrowser : NotificationBase<MediaBrowserSettings>
```

Comment:
It's just too generic. Honestly I don't like the `NzbGet`, `Sabnzbd` classes either.